lib/methods_for_similarity.py

```diff
@@ -0,0 +1,15 @@
+def iou(lista1: list, lista2: list) -> float:
+  # metoda przyjmuje jako argument dwie listy i zwraca ich indeks Jaccarda, czyli liczbę z przedzialu [0,1]
+  if not lista1 or not lista2:
+    return 0
+  else:
+    intersection = []
+    union = []
+    for element in lista1:
+      union.append(element)
+      if element in lista2:
+        intersection.append(element)
+    for element in lista2:
+      if element not in union:
+        union.append(element)
```

Comment:
Nie wymyślajmy koła na nowo
intersec
```suggestion
    intersection = set(lista1).intersection(set(lista2))
    union = set(lista1).union(set(lista2))
```

Reply:
Skomitowałem zasugerowane zmiany